### PR TITLE
feat: friendlier gatsbot stale messages

### LINKED ISSUE #56: Review and edit the stalebot messages to be more inline with the Gatsby "voice"
Right now our stale messages are a bit formal. Let's make them more playful and friendly.

### PR DESCRIPTION
**Pull request by jlengstorf**
This is an attempt to make the stale messaging friendlier and a little more inline with @gatsbot’s personality.

closes #56

### CODE REVIEW

File: tasks/stale.ts [hunk 1]
```diff
@@ -11,9 +11,23 @@ const EXEMPT_LABEL = `not stale`;
...
+
+We get a lot of issues, so we currently close issues after ${DAYS_TO_STALE + DAYS_TO_CLOSE} days of inactivity. It’s been at least ${DAYS_TO_STALE} days since the last update here.
+
+If we missed this issue or if you want to keep it open, please reply here. You can also add the label "${EXEMPT_LABEL}" to keep this issue open!
+
+Thanks for being a part of the Gatsby community! 💪💜`;
+
+const CLOSE_MESSAGE = `Hey again!
+
+It’s been ${DAYS_TO_STALE + DAYS_TO_CLOSE} since anything happened on this issue, so our friendly neighborhood robot (that’s me!) is going to close it.
```
Comment (sidharthachatterjee) [inline tasks/stale.ts]: Possible typo — neighborhood

Comment (sidharthachatterjee) [reply tasks/stale.ts]: Actually never mind, it's an American versus British diff

File: tasks/stale.ts [hunk 2]
```diff
@@ -11,9 +11,23 @@ const EXEMPT_LABEL = `not stale`;
...
+
+Please keep in mind that I’m only a robot, so if I’ve closed this issue in error, I’m so super sorry. Please feel free to reopen this issue or create a new one if there’s anything else you need.
```
Comment (sidharthachatterjee) [inline tasks/stale.ts]: _Maybe_ reads better as this 
```suggestion
Please keep in mind that I’m only a robot, so if I’ve closed this issue in error, I’m so super sorry. Please feel free to reopen this issue or create a new one if you need anything else.
```

Comment (sidharthachatterjee) [reply tasks/stale.ts]: Maybe could remove `please` in the latter sentence since the former also begins with it

Comment (DSchau) [inline tasks/stale.ts]: heh, so take it or leave it, but this reads as sarcastic to me 😛 

```suggestion
Please keep in mind that I’m only a robot, so if I’ve closed this issue in error, I’m so sorry! Please feel free to reopen this issue or create a new one if there’s anything else you need.
```

File: tasks/stale.ts (hunk 1 above)
Comment (jlengstorf) [reply tasks/stale.ts]: Can you expand on this? What’s the typo you see?

Comment (jlengstorf) [reply tasks/stale.ts]: Ah, gotcha.

File: tasks/stale.ts [hunk 3]
```diff
@@ -11,9 +11,23 @@ const EXEMPT_LABEL = `not stale`;
...
+
+If we missed this issue or if you want to keep it open, please reply here. You can also add the label "${EXEMPT_LABEL}" to keep this issue open!
+
+Thanks for being a part of the Gatsby community! 💪💜`;
+
+const CLOSE_MESSAGE = `Hey again!
+
+It’s been ${DAYS_TO_STALE + DAYS_TO_CLOSE} since anything happened on this issue, so our friendly neighborhood robot (that’s me!) is going to close it.
+
+Please keep in mind that I’m only a robot, so if I’ve closed this issue in error, I’m sorry. Please feel free to reopen this issue or create a new one if you need anything else.
```
Comment (m-allanson) [inline tasks/stale.ts]: Is this over the top?

```suggestion
Please keep in mind that I’m only a robot, so if I’ve closed this issue in error, I’m `HUMAN_EMOTION_SORROW`. Please feel free to reopen this issue or create a new one if you need anything else.
```

Comment (jlengstorf) [reply tasks/stale.ts]: I love this.

**Comment by Akryum** [comment]
Hi there! Do you all mind if we used modified or inspired versions of those messages for `vue-bot` covering the `vuejs` org? :ghost:

**Comment by DSchau** [comment]
@Akryum please! Feel free to do so--we're all about sharing and making our separate communities great!

**Pull request merged**